mkdocs.yml

```diff
@@ -2,10 +2,10 @@ copyright: '&copy; 2015-2016'
 extra:
```

Comment:
I suspect this file should be removed from this PR.  It looks like it's is undo'ing changes in `develop`. 

@wwysoc2 would probably know for sure.

Reply:
Good point. @a-collet if you can easily remove mkdocs.yml, please do.  Otherwise, I have a ton to do with the site, so I can add these back.